### ISSUE TITLE
[Bug] geoJSON points not loading

### ISSUE THREAD

**Issue by ford-scoop** [keplergl/kepler.gl#1196]
**Describe the bug**
I am trying to upload geoJSON files that include just a single Feature that is a Point, but am getting a `Failed to upload files` error. 

**To Reproduce**
1. Go to kepler.gl/demo
2. Try to upload this geoJSON snippet (as a .geoJSON file):
```
{
	"type": "Feature", 
	"geometry": {
		"type": "Point", 
		"coordinates": [
			41.8817441, -87.6335398
		]
	}, 
	"properties": {
		"lat": 41.8817441, 
		"lng": -87.6335398, 
		"icon": "place"
	}
}
```

This results in a `Failed to upload files` error. 

**Expected behavior**
It is expected the file uploads and the Point appears on the map. 

**Desktop (please complete the following information):**
 - OS: macOS 10.15.4
 - Browser: chrome
 - Version: 83.0.4103.116 

**Additional context**
This process worked as recently as mid-late May, so I believe a recent release may have broken it. Other geoJSON files that I upload work, though they are all either FeatureCollections or MultiLineStrings.

I also tried to use the examples in the [docs](https://docs.kepler.gl/docs/user-guides/b-kepler-gl-workflow/a-add-data-to-the-map#geojson). The single Feature also failed, while the FeatureCollection uploaded as expected.

Thank you!

**Comment by heshan0131** [comment]
Fix is coming up, for now, use feature collection to wrap single feature. 
Thanks

**Comment by ford-scoop** [comment]
Awesome, thank you! Will do that for the time being :)

**Comment by heshan0131** [comment]
merged with #1212
